Session9/src/com/itschool/homework/Session9Challenge1.java

```diff
@@ -0,0 +1,17 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+
+public class Session9Challenge1 {
```

Comment:
I think these all challenges are from session8, not 9.

Reply:
yes indeed, i was thinking if i should let them inside session9 or 8 but i went straight for the last one, 9. I apologise for the confusion